<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>491</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>525</width>
    <height>491</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>525</width>
    <height>491</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>525</width>
     <height>475</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>525</width>
     <height>450</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">max-width: 525px;
max-height: 450px;</string>
   </property>
   <widget class="QCheckBox" name="chb_HiddingIsOn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>501</width>
      <height>421</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>525</width>
      <height>450</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Управление правилами</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>471</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_AddRule">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить правило</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_ListOfRules">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Список правил</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>350</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Редактировать правило</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>350</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить правило</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_RulesForShow">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Показывать:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtn_AllRules">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>300</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Все правила</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtn_LocalRules">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>300</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Локальные правила</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtn_GlobalRules">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>300</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Глобальные правила</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Просмотреть файлы</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>471</width>
        <height>371</height>
       </rect>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>481</width>
         <height>381</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
